src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Anonymize/Anonymize.ts

```diff
@@ -46,6 +47,7 @@ export var anonymizeDirective = (
                 ]).then((args) => {
                     scope.isOptional = args[1];
                     scope.data.model = scope.isOptional ? adhUser.data.anonymize : false;
+                    scope.data.model = scope.localDefault || scope.data.model;
```

Comment:
IMHO this is not exactly what we want. I see two issues:

-  **`isOptional` has no effect on `localDefault`** (If `isOptional` is false and `localDefault` is true, the result will be true; I would expect false)
-  **`localDefault=false` is ignored** (If `localDefault` is false and `adhUser.data.anonymize` is true, the result will be true; I would expect false)

I propose something like this:

```
var default = typeof scope.localDefault === "undefined" ? adhUser.data.anonymize : scope.localDefault;
scope.data.model = scope.isOptional ? default : false;